db/gen10MData.js

```diff
@@ -1,7 +1,10 @@
 const faker = require('faker');
 const fs = require('fs');
+// const mongoose = require('mongoose');
```

Comment:
Not sure if you're just holding on to them, but delete the comments if you don't need them